components/typography/typography.css

```diff
@@ -0,0 +1,67 @@
+/* Headings */
+
+.heading-1 {
+  font-size: var(--heading-1);
+  font-weight: bold;
+  line-height: var(--line-ht-1);
+  width: 100%;
+}
+
+.heading-2 {
+  font-size: var(--heading-2);
+  font-weight: bold;
+  line-height: var(--line-ht-1);
+  width: 100%;
+}
+
+.heading-3 {
+  font-size: var(--heading-3);
+  font-weight: bold;
+  line-height: var(--line-ht-1);
+  width: 100%;
+}
+
+.heading-4 {
+  font-size: var(--heading-4);
+  font-weight: bold;
+  line-height: var(--line-ht-1);
+  width: 100%;
+}
+
+.heading-5 {
+  font-size: var(--heading-5);
+  font-weight: bold;
+  line-height: var(--line-ht-1);
+  width: 100%;
+}
```

Comment:
Suggestion: these all 5 classes have only one property is different from each other. so you can make one global class for remaining three props and change 4th one in individual class.
So that way your code becomes short and not repeated.

Reply:
I had actually thought of this but didn't implement it because it would increase one class. But now it seems like a good idea to group all the common properties into one class. Thanks